src/content/docs/licenses/license-information/faq/new-relic-one-pricing-plan-frequently-asked-questions.mdx

```diff
@@ -78,9 +78,9 @@ These FAQs are meant to serve as a supplement to the main [New Relic pricing doc
 
   <Collapser
     id="FAQ_8"
-    title="Q: How does the 'Free Tier' impact my pay-as-you-go (PAYG) or Annual Pool of Funds (APOF) subscription?"
+    title="Q: How does the 'Free Tier' impact my pay-as-you-go (PAYG) or contract subscription?"
   >
-    **A:** The Free Tier usage will be deducted automatically from the Monthly Product Usage for Pay-as-you-go or Annual pool of funds subscriptions.
+    **A:** The Free Tier usage will be deducted automatically from the Monthly Product Usage for Pay As You Go or [contract](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#usage-plans) subscriptions.
```

Comment:
Seems like these an inconsistency between "pay-as-you-go" and "Pay As You Go" and I feel like we should pick one or the other.